### PR TITLE
Added security overview page linked to the index page

### PR DESCRIPTION
**Pull request by mattempty**
This page is a high level overview of security of MOJ Forms.

### CODE REVIEW

File: source/documentation/how-we-handle-security.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The whole platform is made from separate services that perform different tasks to collect user data in a structured, secure manner.
+
+MOJ Forms is listed as a Data Asset by the MOJ DPIA team, which means the MOJ Forms part of the DPIA process is already assured and only needs referencing. 
+
+## Editor
+The Editor is a staff facing application which provides the tools to build a form which meets the user journey. Add settings and publish to a test environment and then publish to the live environment.
+
+The Form uses json to store the content and branching instructions. This metadata can be stored in the open as it does not contain anything confidential or personal.
+
+Accessing the Editor we use Single Sign On which confirms the authentication with our domains Active Directories.
```
Comment (StevenLeighton21) [inline source/documentation/how-we-handle-security.html.md.erb:21]: Maybe mention provider?

File: source/documentation/how-we-handle-security.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+5. Submitter - Passes the data to the backend team as directed in the settings for the form.
+
+### Service Token Cache
+This service is used to confirm each application is communicating with the application within the platform. This fulfils one of the principles to make sure we do not reply to the hosting for security between the services. For example, the runner and the user datastore will request a token and only if they match will they communicate.
+
+More information can be found in the [platforms](/#moj-forms-platform) section.
+
+### End User data
+The data is transported over https and encrypted using the session cookie generated when the user starts the form so it is stored encrypted per-user + per-form. This Sequential data is encrypted at rest in an RDS hosted in AWS.
+Documents are checked on upload that they meet the type and size validation and streamed to a malware checker, only then it is stored encrypted in a S3 bucket. Only the internal services can add and get files from the filestore.
```
Comment (StevenLeighton21) [inline source/documentation/how-we-handle-security.html.md.erb:39]: Might mention that bucket connections are https and bucket contents are encrypted at rest specifically?

File: source/documentation/how-we-handle-security.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+4. Malware Scanner - Scans uploaded documents for malware
+5. Submitter - Passes the data to the backend team as directed in the settings for the form.
+
+### Service Token Cache
+This service is used to confirm each application is communicating with the application within the platform. This fulfils one of the principles to make sure we do not reply to the hosting for security between the services. For example, the runner and the user datastore will request a token and only if they match will they communicate.
+
+More information can be found in the [platforms](/#moj-forms-platform) section.
+
+### End User data
+The data is transported over https and encrypted using the session cookie generated when the user starts the form so it is stored encrypted per-user + per-form. This Sequential data is encrypted at rest in an RDS hosted in AWS.
```
Comment (StevenLeighton21) [inline source/documentation/how-we-handle-security.html.md.erb:38]:
```suggestion
The data is transported over https and encrypted at rest using the session cookie generated when the user starts the form so it is stored encrypted per-user + per-form. This Sequential data is encrypted at rest in an RDS hosted in AWS. We do not store or even read the session keys, and user data is deleted after 28 days automatically.
```

**Review by H3ll3m4 — APPROVED** [review]
🚀

**Review by StevenLeighton21 — APPROVED** [review]

**Review by H3ll3m4 — APPROVED** [review]
🚀

**Pull request merged**